pkg/ruler/ruler.go

```diff
@@ -92,70 +68,84 @@ type Config struct {
 // RegisterFlags adds the flags required to config this to the given FlagSet
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	cfg.LifecyclerConfig.RegisterFlagsWithPrefix("ruler.", f)
+	cfg.StoreConfig.RegisterFlags(f)
+
+	// Deprecated Flags that will be maintained to avoid user disruption
+	flagext.DeprecatedFlag(f, "ruler.client-timeout", "This flag has been renamed to ruler.configs.client-timeout")
```

Comment:
`ruler.configs.url` appears to be missing from this list of deprecated flags. It was part of the deleted `pkg/configs/client/config.go` file.

Reply:
This flag also still exists. It got move a bit and is registered in `pkg/ruler/storage.go` https://github.com/cortexproject/cortex/pull/1571/files#diff-16c509ab46b783eb193e10999f09ed31R21